graf2d/postscript/inc/TImageDump.h

```diff
@@ -21,8 +21,8 @@ class TPoint;
 
 class TImageDump : public TVirtualPS {
 protected:
-   TImage           *fImage;     ///< Image
-   Int_t             fType;      ///< PostScript workstation type
+   TImage           *fImage{nullptr};     ///< Image
+   Int_t             fType{0};            ///< PostScript workstation type
```

Comment:
I know fType is initialized elsewhere, but does it really arm to have it here also?

Reply:
If constructor has direct assignment of that member, than default value will be overwritten.